niceday-router/README.md

```diff
@@ -0,0 +1,28 @@
+# Niceday Router
+
+## Overview
+This application listens to incoming messages from the Niceday server.
+When a message comes in from a user sent to the therapist account, 
+the message is forwarded to the rasa agent rest API. 
+The rasa response is then sent back to the user in the Niceday app. 
+
+### Setup
+1. Install npm and node-js, following [these instructions](https://www.npmjs.com/get-npm).
+2. Install dependencies: Run `npm install`.
+3. Run `node login.mjs emailaddress password`, 
+where emailaddress and password should be the credentials for the therapist account. 
+This will print the user id of the therapist as well as the token that you need to authenticate.
+4. Create a file called `.env` in the root of this app.
```

Comment:
Add that the token expires!

Reply:
If token is expired, `npm start` will give a `ChatNotAuthorizedError`.